pulpcore/tests/functional/api/using_plugin/test_pulpimport.py

```diff
@@ -236,3 +298,118 @@ def test_chunked_import(self):
         for repo in self.import_repos:
             repo = self.repo_api.read(repo.pulp_href)
             self.assertEqual(f"{repo.pulp_href}versions/1/", repo.latest_version_href)
+
+    def test_import_check_valid_path(self):
+        body = {"path": self._find_path()}
+        result = self.import_check_api.pulp_import_check_post(body)
+        self.assertEqual(result.path.context, self._find_path())
+        self.assertTrue(result.path.is_valid)
+        self.assertEqual(len(result.path.messages), 0)
+        self.assertIsNone(result.toc)
+        self.assertIsNone(result.repo_mapping)
+
+    def test_import_check_valid_toc(self):
+        body = {"toc": self._find_toc()}
+        result = self.import_check_api.pulp_import_check_post(body)
+        self.assertEqual(result.toc.context, self._find_toc())
+        self.assertTrue(result.toc.is_valid)
+        self.assertEqual(len(result.toc.messages), 0)
+        self.assertIsNone(result.path)
+        self.assertIsNone(result.repo_mapping)
+
+    def test_import_check_repo_mapping(self):
+        body = {"repo_mapping": json.dumps({"foo": "bar"})}
+        result = self.import_check_api.pulp_import_check_post(body)
+        self.assertEqual(result.repo_mapping.context, json.dumps({"foo": "bar"}))
+        self.assertTrue(result.repo_mapping.is_valid)
+        self.assertEqual(len(result.repo_mapping.messages), 0)
+        self.assertIsNone(result.path)
+        self.assertIsNone(result.toc)
+
+        body = {"repo_mapping": '{"foo": "bar"'}
+        result = self.import_check_api.pulp_import_check_post(body)
+        self.assertEqual(result.repo_mapping.context, '{"foo": "bar"')
+        self.assertFalse(result.repo_mapping.is_valid)
+        self.assertEqual(result.repo_mapping.messages[0], "invalid JSON")
+
+    def test_import_check_not_allowed(self):
+        body = {"path": "/notinallowedimports"}
+        result = self.import_check_api.pulp_import_check_post(body)
+        self.assertEquals(result.path.context, "/notinallowedimports")
```

Comment:
Pytest says assertEquals is deprecated.
```suggestion
        self.assertEqual(result.path.context, "/notinallowedimports")
```

Reply:
Yeah, looks like I missed a couple of those - been using assertEquals() for too long.